### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by dryabov**
### Summary of Changes
Add new Joomla!4-style UI

**Note#1**: merge it AFTER the PR #85 only.

**Note#2**: the ordering of rules is now set in the `views/uploads/view.html.php` file, and new rules has to be added there. Probably, if many additional rules are expected, it would be better to replace this list by a `priority` or `ordering` field in the `JEDcheckerRule` class, and sort list of rules accordingly.

**Note#3**: `j4-style.css` file (for Joomla!3) is generated from J!4.b7's `atum` template by using `purgecss` (to keep necessary selectors only) and some manual postprocessing. 

### Testing Instructions
Install and check

### Actual result BEFORE applying this Pull Request
Old Joomla!3-style UI is used.

### Expected result AFTER applying this Pull Request
New Joomla!4-style UI is used.

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/components/com_jedchecker/views/uploads/view.html.php [hunk 1]
```diff
@@ -44,13 +50,33 @@ public function display($tpl = null)
 	 */
 	public function getRules()
 	{
-		$rules = array();
-		$files = JFolder::files(JPATH_COMPONENT_ADMINISTRATOR . '/libraries/rules', '\.php$', false, false);
+		$existingRules = array(
```
Comment (Llewellynvdm) [inline administrator/components/com_jedchecker/views/uploads/view.html.php:53]: Why are you hard coding this? it causes a serious headache in maintainability.

**Comment by Llewellynvdm** [comment]
#### Maybe I am not saying this enough, but I am and I can even say we are very grateful for all your contribution, it is absolutely amazing, thank you!

It is a pity that most of my response to your work has been corrective in nature, please remember that I am very happy that you are contributing, and mostly find your code very interesting and informative. Well done!

File: administrator/components/com_jedchecker/views/uploads/view.html.php (hunk 1 above)
Comment (dryabov) [reply administrator/components/com_jedchecker/views/uploads/view.html.php:53]: To arrange them in specific order (for sidebar menu). Alternative way is to introduce a `priority` static property, and sort according to that property, but I've postponed this idea until later. I'll try to implement it in a few days.

**Comment by dryabov** [comment]
@Llewellynvdm Thank you. Discussion is normal process here.
"Discussio mater veritas est" (*The discussion is the mother of truth*, sometimes translated as *Thought thrives on conflict*, Sokrates).

File: administrator/components/com_jedchecker/views/uploads/view.html.php (hunk 1 above)
Comment (dryabov) [reply administrator/components/com_jedchecker/views/uploads/view.html.php]: Done

**Comment by dryabov** [comment]
@Llewellynvdm Done

**Pull request merged**